src/shared/Database/Database.h

```diff
@@ -267,7 +267,7 @@ class Database
         SqlDelayThread*     m_threadBody;                   ///< Pointer to delay sql executer (owned by m_delayThread)
         MaNGOS::Thread*     m_delayThread;                  ///< Pointer to executer thread
 
-        bool m_bAllowAsyncTransactions;                     ///< flag which specifies if async transactions are enabled
+        std::atomic<bool> mAllowAsyncTransactions;          ///< flag which specifies if async transactions are enabled
```

Comment:
I think the current codestyle is "m_allowAsyncTransactions.

Reply:
Ah, good. I was in doubt about that after seeing various styles being used - though often in old(er) code. I checked the wiki page with the coding standard and it wasn't entirely clear about that, in the sense that the example class shows a member ```m_field``` but the text never mentions anything about it.

I'll change this, also in future changes, no problem.